examples/nullchain.py

```diff
@@ -1,11 +1,14 @@
 import logging
-import numpy as np
 from collections import namedtuple
+from random import randint
 
 from vega_sim.null_service import VegaServiceNull
-import vega_sim.proto.vega as vega_protos
-from vega_sim.proto.vega.governance_pb2 import UpdateMarketConfiguration
+from vega_sim.service import MarketStateUpdateType
 
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-p", "--perps", action="store_true", default=False)
```

Comment:
This part should be inside the `__name__` part (doesn't hugely matter since it's just an example file but still)

Reply:
Sure, moved now.